src/js/entities-service/clinicians.js

```diff
@@ -21,8 +22,12 @@ const Entity = BaseEntity.extend({
   },
   fetchByWorkspace(workspaceId) {
     const url = `/api/workspaces/${ workspaceId }/relationships/clinicians`;
+    const workspace = Radio.request('entities', 'workspaces:model', workspaceId);
 
-    return this.fetchCollection({ url });
+    return this.fetchCollection({ url })
+      .then(clinicians => {
+        workspace.updateClinicians(clinicians);
```

Comment:
this makes more sense here but is essentially only necessary for added/disabled clinicians.  nearly unnecessary.